### ISSUE TITLE
Pulse card render error: Output of renderable-interval does not match schema

### ISSUE THREAD

**Issue by nileshtrivedi** [metabase/metabase#10326]
**Describe the bug**
Unable to render a card while creating a pulse

**Logs**
```
07-11 06:38:06 ERROR pulse.render :: Pulse card render error
clojure.lang.ExceptionInfo: Output of renderable-interval__34238 does not match schema: 

	   {:interval-start missing-required-key, :interval missing-required-key, :this-interval-name missing-required-key, :last-interval-name missing-required-key, :ns-str disallowed-key, :msg disallowed-key, :args disallowed-key}  

 {:type :schema.core/error, :schema {:interval-start org.joda.time.DateMidnight, :interval org.joda.time.base.BaseSingleFieldPeriod, :this-interval-name metabase.util.i18n.UserLocalizedString, :last-interval-name metabase.util.i18n.UserLocalizedString}, :value "Last month", :error {:interval-start missing-required-key, :interval missing-required-key, :this-interval-name missing-required-key, :last-interval-name missing-required-key, :ns-str disallowed-key, :msg disallowed-key, :args disallowed-key}}
	at metabase.pulse.render.datetime$eval34239$renderable_interval__34238__34249.invoke(datetime.clj:74)

```

**To Reproduce**
1. Run `lein ring server`
2. Create a question on sample dataset (count of orders grouped by month). See screenshot.
3. Try to configure a pulse with this question
4. See error: "An error occured while displaying this card"

**Screenshots**
![Screenshot from 2019-07-11 06-43-46](https://user-images.githubusercontent.com/19304/61014995-9c615080-a3a7-11e9-93ee-ba470653bbfe.png)

**Information about your Metabase Installation:**
- Metabase version: v0.33.0-preview2. Also able to reproduce on current master.
- With H2 database

**Comment by mazameli** [comment]
I reproduced this on master as well as `notebook-mode` (which is v0.33.0-preview2 plus some more commits), but in my testing this **only happens with saved questions that are a single-metric time series**, which get rendered as a spark line in pulses. Questions that have _two_ metrics broken out by time render just fine for me, because they render as a table. All other question types seem to render fine for me as well. Does this match with your experiences, @nileshtrivedi?

**Comment by eshnil** [comment]
Yes, it was only for specific type of questions. And I verified the fix. Works fine in latest master. :+1:

**Comment by salsakran** [comment]
Hello all, this is fixed in 0.32.10 which just went out.
